licenses-binary/LICENSE-public-domain.txt

```diff
@@ -0,0 +1,4 @@
+Text copied verbatim from http://aopalliance.sourceforge.net/
```

Comment:
Which library is this for? `org.glassfish.hk2.external:aopalliance-repackaged` seems to be under EPL.